docs/src/rfc/active/001-customization-overhaul.md

```diff
@@ -0,0 +1,164 @@
+      3. Crafting gear takes a huge amount of downtime with so-so roleplaying impact
+      4. Ease of swapping increases ability to hard counter specific encounters, particularly when combined with free selection
+      5. Getting new gear is not very impactful due to already-optimized builds
+      6. Gear and essence crystal economy is very important due to power budget tied up in gear
+   4. Miscellaneous
+      1. Half-point affixes are clunky
+      2. Problems with gear cascade to monster building
+      3. Free selection of affixes limits player interest in testing out alternative affixes in a niche they care about
+10. Consumable choice
+    1. Complexity, character identity and balancing issues of gear, but weaker
+    2. "Signature consumable" pattern is not supported very well
+    3. Discovering new consumables isn't exciting due to free selection of affixes
+
+The rolling respec on level-up is a fantastic addition to almost all of these choices, and helps them feel like part of your character without being truly permanent.
+As a result, gear feels both more persistent than other character customizations because it costs a scarce resource (downtime activities) to create, but also less persistent, since you can tweak it as many times as you'd like or instantly pick up a new piece of gear.
+
+## Sub-proposal: Shift budget from gear to character customization
+
+### Guide-level explanation
+
+Most of your character's customization (and power) comes from their character choices, rather than the gear they find or make.
+By selecting **feats**.
+
+You can select any **feat** from your class and species features when you gain one due to levelling up.
+These interact directly with your innate features, while **gear affixes** are simpler complements to your build,
+intended to help you adapt to the situation at hand and work for any class and species.
+
+### Reference-level explanation
+
+1. Combat talents are removed, and replaced with feats.
+2. Skill talents are renamed to **knacks**.
+3. Gear affix point budget is scaled down.
+4. Gain one feat at level 1, and one more every ~4 levels (when your attributes increase).
+5. Feats are class and species specific; you can select from either list.
+6. When you're resurrected as a new species, you must replace all species-specific feats with ones from your new species.
+
+### Drawbacks
+
+1. Another customization system to learn
+2. Requires creation of class / species specific content
+   1. More work
+   2. More space, but see the "effect cards" subproposal below
+   3. Increases barrier to homebrew
+3. Requires significant balancing and design skill to account for possible interactions between feats
+4. Mild analysis paralysis when choosing feats
+5. No more "dead-simple" level-ups
+
+### Open questions
+
+1. Is "knacks" a good name to replace skill talents?
+2. What does the new gear and feat scaling structure look like?
+3. Should we allow sub-species / sub-class specific feats?
```

Comment:
Yes. This pattern will mostly be seen with sub-species, but there may be sub-classes that would like it too. I'd stray away from sub-class feats to avoid complexity though.